types/keymaster/keymaster-tests.ts

```diff
@@ -1,14 +1,19 @@
-
-
 // simple key binding without scope
-key('shift+a', function (keyboardEvent: KeyboardEvent, keymasterEvent: KeymasterEvent) {
+key("shift+a", (keyboardEvent: KeyboardEvent, keymasterEvent: KeymasterEvent) => {
```

Comment:
Sorry, overlooked this:
```suggestion
key("shift+a", (keyboardEvent, keymasterEvent) => {
```